TestsExample/ios/Podfile.lock

```diff
@@ -514,4 +514,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 123196e881422a7a7f3a9c228d6113e8de498360
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.10.0
```

Comment:
We shouldn't mix normal changes with such ones in the same PR. Please change it back to what it was.